models/signUpDao.js

```diff
@@ -0,0 +1,29 @@
+const { myDataSource } = require("./myDataSource");
+
+//search if there is a same email address
+const emailCheck = async (email) => {
+  const [result] = await myDataSource.query(
+    `
+    SELECT 
+    email 
+    FROM users 
+    WHERE email = ?`,
+    [email]
+  );
+  return !result ? true : false;
+};
+
+const signUp = async (name, email, hashedPassword, phoneNumber) => {
+  try {
+    return await myDataSource.query(
+      `INSERT INTO users (name, email, password, phone_number) VALUES (?,?,?,?);`,
+      [name, email, hashedPassword, phoneNumber]
+    );
+  } catch (err) {
+    console.log(err);
+    const error = new Error("Unknown ERROR during signUp");
+    error.statusCode = 500;
```

Comment:
status code 500은 백엔드에서 처리하지 못 한 error를 타나냅니다.
400 정도로 수정해 주면 좋을 것 같아요.
```suggestion
    error.statusCode = 400;
```